app/views/pages/home.html.erb

```diff
@@ -0,0 +1,36 @@
+<h1 class="govuk-heading-l">Register trainee teachers</h1>
```

Comment:
I think this heading, and the text and button below should all be inside a `govuk-grid-row` and a `govuk-grid-column-two-thirds-from-desktop`, so that they're not full width.

Reply:
Yes, all this should be in a column as @AbigailMcP says.